app/graph/types/team_type.rb

```diff
@@ -400,4 +400,56 @@ def statistics(period:, language: nil, platform: nil)
     return nil unless User.current&.is_admin
     TeamStatistics.new(object, period, language, platform)
   end
+
+  field :bot_query, [TiplineSearchResultType], null: true do
+    argument :search_text, GraphQL::Types::String, required: true
+  end
+
+  def bot_query(search_text:)
+    unless User.current&.is_admin
+      raise GraphQL::ExecutionError, "You do not have permission to perform this action"
+    end
+
+    return nil unless search_text
+
+    explainers = object.explainers.order(created_at: :desc).limit(3)
+
+    fact_checks = FactCheck.joins(:claim_description)
+                           .where(claim_descriptions: { team_id: object.id })
+                           .order('fact_checks.created_at DESC')
+                           .limit(3)
```

Comment:
Suggestion: Order by `updated_at` instead of `created_at` - it gives the frontend team more flexibility on how to prepare their data locally.